### DISCUSSION TITLE
RunTimeError when using hep.cms.label()

### DISCUSSION THREAD

**Discussion by Wilsker** [scikit-hep/mplhep#298] [Q&A]
Hello!

I am getting a runtime error when I try to implement hep.cms.label() in some code. Please find beneath a simple script exemplifying the issue.

Thanks,

Josh

```
# Python 3.9.4
# matplotlib 3.4.2
# mplhep  0.3.4

import matplotlib
import matplotlib.pyplot as plt
import mplhep as hep

def main():
    plt.style.use([hep.style.ROOT, hep.style.firamath])
    x_values = [3.9, 4.1, 3.4, 4.7]
    y_values = [0., 1., 2., 3.]
    fig, ax = plt.subplots()
    ax.scatter(x_values, y_values, label='XXX')
    hep.cms.label(loc=0) # problematic line
    plt.savefig('issue_plot.png')
    return

if __name__ == '__main__':
    main()
    exit(0)

```

**Comment by andrzejnovak** [comment]
Issue with MaxOSX backend. Setting `matplotlib.use('TkAgg')` should fix it.

More info at:
https://stackoverflow.com/questions/30138026/seaborn-matplotlib-cannot-get-window-extent-w-o-renderer-runtimeerror
